src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -1837,6 +1837,45 @@ def test_aks_create_add_nodepool_with_custom_ca_trust_certificates(self, resourc
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_nodepool_node_soak_duration(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=81489510&view=results) to validate the change.

Reply:
Thanks @FumingZhang! Checking why it failed.

Btw

```
echo 'Live test failed!'
echo 'Please refer to this wiki (https://dev.azure.com/msazure/CloudNativeCompute/_wiki/wikis/CloudNativeCompute.wiki/156735/AZCLI-AKS-Live-Unit-Test-Pipeline) for troubleshooting guidelines.'
```

Looks like that wiki link is outdated. I updated the `WIKI_LINK` pipeline variable [here](https://dev.azure.com/msazure/CloudNativeCompute/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=219428&view=Tab_History), was that the correct url?

Reply:
@FumingZhang so, it looks like the live test is currently failing

> Query 'upgradeSettings.nodeSoakDurationInMinutes' doesn't yield expected value '5', instead the actual value is 'None'

I noticed that your [2023-09-02-preview](https://github.com/Azure/azure-rest-api-specs/tree/FumingZhang-containerservice-Microsoft.ContainerService-2023-09-02-preview) branch (which contains the [Swagger change](https://github.com/Azure/azure-rest-api-specs/pull/26046) for this feature) in Azure/azure-rest-api-specs hasn't been merged yet - I guess that explains it? Let's re-run the test after the Swagger change is merged.

Reply:
Yes, swagger/sdk has not been updated, so the deserialized response will not include this property.

Reply:
SDK not updated yet, rebase/merge from main once this PR #6904 got merged

Reply:
Thx - looks like that PR just got merged.

[synced/merged](https://github.com/MaxHorstmann/azure-cli-extensions/commit/065f4a9c44de9381fcbb9128d31da0c2c323cd08) this branch.

queued up new [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=81902688&view=results)

Reply:
Seems live test passed, please commit the recording files corresponding to the newly added test cases (find from pipeline artifact).

Please also fix errors in [Static Analysis](https://dev.azure.com/azclitools/public/_build/results?buildId=100166&view=logs&j=ce14e7bd-b9ee-5bd7-1dc4-cb4d70edaf0d&t=90fe1415-141e-56cb-5eb9-d53a9bbb8155&l=3574).

Reply:
> Seems live test passed, please commit the recording files corresponding to the newly added test cases (find from pipeline artifact)

[done](https://github.com/Azure/azure-cli-extensions/pull/6865/commits/1b1eca343377e206efa9b26ae8e34f73ff3a3324)

> Please also fix errors in [Static Analysis](https://dev.azure.com/azclitools/public/_build/results?buildId=100166&view=logs&j=ce14e7bd-b9ee-5bd7-1dc4-cb4d70edaf0d&t=90fe1415-141e-56cb-5eb9-d53a9bbb8155&l=3574).

[done](https://github.com/Azure/azure-cli-extensions/pull/6865/commits/67055f210b96f20ae0d034399e80141aed4452d2)